src/backend/function/Fusion.Resources.Functions/Functions/Notifications/ResourceOwner/WeeklyReport/ScheduledReportTimerTriggerFunction.cs

```diff
@@ -83,41 +88,32 @@ private async Task SendResourceOwnersToQueue(ServiceBusSender sender, int totalB
             // Query departments from LineOrg
             var departments = (await _lineOrgClient.GetOrgUnitDepartmentsAsync())
                 .Where(d => d.FullDepartment != null)                     // Exclude departments with blank department name
-                .Where(d => d.FullDepartment.Contains("PRD"))
-                .Where(x => x.Management?.Persons.Length > 0);            // Exclude departments with no receivers
-
-            // Group OrgUnits by FullDepartment and join the Person arrays together
-            var groupedDepartments = departments
-                .GroupBy(orgUnit => orgUnit.FullDepartment)
-                .Select(group =>
-                {
-                    // Combine the Person arrays of all OrgUnits in the group into a single array
-                    var allPersons = group.SelectMany(orgUnit => orgUnit.Management.Persons).ToArray();
-
-                    // Create a new OrgUnits object with the FullDepartment, SapId, and combined Person array
-                    return new OrgUnits
-                    {
-                        FullDepartment = group.Key,
-                        SapId = group.First().SapId,
-                        Management = new Management { Persons = allPersons }
-                    };
-                })
+                .Where(d => d.FullDepartment.Contains("PDP PRD PMC PCA PCA5"))
```

Comment:
For testing puposes? Should remove this before merging

Reply:
Fixed